scalardb-test/grpc-config.toml.tmpl

```diff
@@ -0,0 +1,32 @@
+[modules]
+  {{ if (isTrue (default .Env.INIT_ENABLED "true")) }}
+  [modules.preprocessor]
+    name = "kelpie.scalardb.transfer.TransferPreparer"
+    path = "build/libs/scalardb-test-all.jar"
+  {{ end }}
+  {{ if (isTrue (default .Env.PROCESS_ENABLED "true")) }}
+  [modules.processor]
+    name = "kelpie.scalardb.transfer.TransferProcessor"
+    path = "build/libs/scalardb-test-all.jar"
+  {{ end }}
+  [modules.postprocessor]
+    name = "kelpie.scalardb.transfer.TransferReporter"
+    path = "build/libs/scalardb-test-all.jar"
+
+[common]
+  concurrency = 1
```

Comment:
We assume one concurrency (thread) per one container, right? If so, we need to increase the number of containers if we want to increase the concurrency, correct?

Reply:
@brfrn169 
You are right.
I set it for grpc config with the above assumption.
Would it be better to be able to set `concurrency` as an environment variable to support other use cases?

Reply:
Thank you! Maybe it would be better to be able to set `concurrency` as an environment variable just in case.

I usually run the kelpie test in 64 concurrencies or something like that. In that case, if we need to start 64 containers, I feel it's a little bit expensive. That's why I think it's better to make `concurrency' configurable. What do you think?

Reply:
I understanding. :bow:
The following added support for setting environment variables.

https://github.com/scalar-labs/kelpie-test/pull/41/commits/d1299df3e76da7531529545c0af111a3a1e9b470